Author: Sven Joachim <svenjoac@gmx.de>
Description: Fix tabs.1 manpage installation
 This reverts a broken change from the 20190623 upstream patchlevel
 which installed /usr/share/man/man1/tabs.1 as a dangling symlink.
Bug: https://lists.gnu.org/archive/html/bug-ncurses/2019-07/msg00022.html
Forwarded: https://lists.gnu.org/archive/html/bug-ncurses/2019-07/msg00023.html
Last-Update: 2019-07-17

---
 man/man_db.renames |    1 -
 1 file changed, 1 deletion(-)

--- a/man/man_db.renames
+++ b/man/man_db.renames
@@ -171,7 +171,6 @@ user_caps.5			user_caps.5
 wresize.3x			wresize.3ncurses
 #
 # Other:
-tabs.1m				tabs.1
 tack.1m				tack.1
 #
 getty.1				getty.8
